theia-php-docker/next.package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia PHP Example",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/callhierarchy": "next",
+        "@theia/editor-preview": "next",
+        "@theia/file-search": "next",
+        "@theia/getting-started": "next",
+        "@theia/git": "next",
+        "@theia/json": "next",
+        "@theia/markers": "next",
+        "@theia/merge-conflicts": "next",
+        "@theia/messages": "next",
+        "@theia/mini-browser": "next",
+        "@theia/navigator": "next",
+        "@theia/outline-view": "next",
+        "@theia/plugin-ext-vscode": "next",
+        "@theia/preferences": "next",
+        "@theia/preview": "next",
+        "@theia/search-in-workspace": "next",
+        "@theia/terminal": "next",
+        "@theia/textmate-grammars": "next"
+    },
+    "devDependencies": {
+        "@theia/cli": "next",
+        "@theia/debug": "next"
+    },
+    "scripts": {
+        "postinstall": "download-debug-adapters"
+    },
+    "adapterDir": "plugins",
+    "adapters": {
+        "php-intellisense": "https://ms-vscode.gallery.vsassets.io/_apis/public/gallery/publisher/felixfbecker/extension/php-intellisense/2.3.5/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage", 
+        "php-debug": "https://ms-vscode.gallery.vsassets.io/_apis/public/gallery/publisher/felixfbecker/extension/php-debug/1.13.0/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage"
```

Comment:
Can we replace the `php-debug` to directly use the extension from the repository rather than the marketplace.
Legally we cannot directly access the marketplace in this manner any longer.

https://github.com/felixfbecker/vscode-php-debug/releases

Reply:
TBC, you should point to the URL of a specific `.vsix` file, not just the release URL.